config/L1T_workspace_slim.json

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
Why are we adding a slim version? Is this a shifter specific list?

Reply:
Ah I see. If you think it is useful for shifters, I think having it in is useful in order to get feedback.